int16 speed
int16 radius
---
int8 status

